src/planner_utils.c

```diff
@@ -7,7 +7,8 @@
 static void plantree_walker(Plan **plan, void (*walker) (Plan **, void *), void *ctx);
 
 static inline void
-plantree_walk_subplans(List *plans, void (*walker) (Plan **, void *), void *ctx)
+plantree_walk_subplans(List *plans,
```

Comment:
What's going on here? Is this some diff in how `pgindent` works?

Reply:
so my pgindent indents this function here if I don't add this newline. Yours doesn't?